concourse/steps/component_descriptor.mako

```diff
@@ -170,11 +170,21 @@ base_descriptor_file_v2 = os.path.join(descriptor_out_dir, 'base_component_descr
 with open(base_descriptor_file_v2, 'w') as f:
   f.write(dump_component_descriptor_v2(base_descriptor_v2))
 
+# dump base_archive_v2 as valid component archive and pass it to descriptor script
```

Comment:
nit: did we not call this ctf-archive, or sth. like that?

Reply:
yes there is a `CTF_PATH` but thats for the output. I wanted to add something similar to the base component descriptors so that if its used it shoiuld be copied to `CTF_PATH`.
I can also name it accordingly `BASE_CTF_PATH`